backend/validation/format.js

```diff
@@ -86,6 +88,50 @@ var verifyAdminId = id => {
   return true;
 };
 
+/**
+ * Verifies that the curriculum name only contains alphanumerical characters
+ * @param {string} curriculum
+ */
+var verifyCurriculumName = curriculum => {
+  if (!curriculum || String(curriculum).length > MAX_CURR_NAME_LENGTH) {
+    return false;
+  }
+  return isAlphanumeric(username);
```

Comment:
My fault, you couldnt have known so not your fault, I wrote some curriculums with like a - in between words., just a heads up, is there a way to include the - symbol ?

Reply:
I'll look into it